### PR TITLE
fix: use closedAt timestamp instead of epoch

### PR DESCRIPTION
**Pull request by gusinacio**

### CODE REVIEW

File: common/src/allocations/monitor.rs [hunk 1]
```diff
@@ -111,12 +81,13 @@ pub fn indexer_allocations(
...
-                .map_err(|e| format!("Failed to fetch current epoch: {}", e))?;
-
-            // Allocations can be closed one epoch into the past
-            let closed_at_epoch_threshold = current_epoch - 1;
+            // Allocations can be closed 5 mins into the past
+            let start = SystemTime::now();
+            let since_the_epoch = start
+                .duration_since(UNIX_EPOCH)
+                .expect("Time went backwards");
+            const MINUTES_THRESHOLD: u64 = 5;
```
Comment (aasseman) [inline common/src/allocations/monitor.rs:89]: Perhaps are safer default could be 1 hour? (yes sorry I know I gave the example of 5min earlier today).
Could it make sense to have this a program parameter?

Comment (gusinacio) [reply common/src/allocations/monitor.rs:89]: Well, in that case we'd need to have a parameter for both indexer-service and tap-agent.

Comment (gusinacio) [reply common/src/allocations/monitor.rs]: Just added a config for both indexer-service and tap-agent. The default value is 1 hour.

File: tap-agent/src/agent.rs [hunk 2]
```diff
@@ -76,8 +74,8 @@ pub async fn start_agent() -> (ActorRef<SenderAccountsManagerMessage>, JoinHandl
     let indexer_allocations = indexer_allocations(
         network_subgraph,
         *indexer_address,
-        GRAPH_NETWORK_ID,
         Duration::from_millis(*allocation_syncing_interval_ms),
+        Duration::from_millis(*recently_closed_allocation_buffer_seconds),
```
Comment (aasseman) [inline tap-agent/src/agent.rs:78]:
```suggestion
        Duration::from_secs(*recently_closed_allocation_buffer_seconds),
```

File: common/src/allocations/monitor.rs [hunk 3]
```diff
@@ -111,12 +82,12 @@ pub fn indexer_allocations(
     // Refresh indexer allocations every now and then
     timer(interval).map_with_retry(
         move |_| async move {
-            let current_epoch = current_epoch(network_subgraph, graph_network_id)
-                .await
-                .map_err(|e| format!("Failed to fetch current epoch: {}", e))?;
-
-            // Allocations can be closed one epoch into the past
-            let closed_at_epoch_threshold = current_epoch - 1;
+            // Allocations can be closed 5 mins into the past
```
Comment (aasseman) [inline common/src/allocations/monitor.rs:85]:
```suggestion
            // Allocations are eligible even if closed for up to `recently_closed_allocation_buffer` 
```

**Review by aasseman — CHANGES_REQUESTED** [review]
LGTM, just a couple little things

**Review by aasseman — APPROVED** [review]

**Pull request merged**